### DISCUSSION TITLE
The formula in Google Sheets with a comma?

### DISCUSSION THREAD

**Discussion by YaroslavTsvirkun** [demmings/gsSQL#59] [Q&A]
Hello. How do you write a formula in Google Sheets with a comma?

**Comment by VegaCaroMiguel** [comment]
Hi @YaroslavTsvirkun:
If you want to use a comma as a function argument separator, you can change the function argument separator setting in Google Sheets:

Go to File > Spreadsheet settings.
Under the "General" tab, find the "Locale" section.
Change the "Function argument separator" setting to the desired character (comma or semicolon).
Click Save settings.
After changing this setting, you can write formulas with commas as argument separators. For example:
=SUM(A1,A10)

Give It a shot, 
Have a nice one !

**Comment by demmings** [comment]
Hi YaroslavTsvirkun and VegaCaroMiguel.

Thanks for your comments.  If you want, let me know how you are using the function (if at all) and how it is working out for you.

Thanks Chris.
